llvm/lib/Transforms/Scalar/LICM.cpp

```diff
@@ -218,6 +218,8 @@ using PointersAndHasReadsOutsideSet =
 static SmallVector<PointersAndHasReadsOutsideSet, 0>
 collectPromotionCandidates(MemorySSA *MSSA, AliasAnalysis *AA, Loop *L);
 
+extern cl::opt<bool> ProfcheckDisableMetadataFixes;
```

Comment:
This needs to be in the header, with LLVM_ABI. Otherwise this will break the dylib build on Windows.